indico/modules/events/papers/controllers/management.py

```diff
@@ -51,3 +54,32 @@ def _process(self):
             logger.info("Paper managers of %r updated by %r", self.event_new, session.user)
             return jsonify_data(flash=False)
         return jsonify_form(form)
+
+
+class RHSwitchReviewingType(RHManagePapersBase):
+
+    CSRF_ENABLED = True
+
+    def _process_PUT(self):
+        event = self.event_new
+        reviewing_type = request.view_args['reviewing_type']
+        if reviewing_type not in ('content', 'layout'):
+            raise Forbidden('Reviewing type not available')
+        event.cfp.toggle_reviewing(reviewing_type=reviewing_type, enable=True)
```

Comment:
I don't think we need kwargs style here - it's quite obvious what the arguments do :)